mozilla-release/mobile/android/app/build.gradle

```diff
@@ -18,6 +18,7 @@ android {
         minSdkVersion project.ext.minSdkVersion
         manifestPlaceholders = project.ext.manifestPlaceholders
 
+        versionCode 94
```

Comment:
Didnt we get this value from the config file?

Reply:
It was not possible to do that because setting that in the config file makes it the default versionCode